procedural/examples/foo-procedures/src/main/java/gov/nasa/ammos/aerie/procedural/examples/fooprocedures/procedures/SampleProcedure.java

```diff
@@ -19,9 +21,12 @@ public void run(@NotNull final EditablePlan plan) {
     var currentTime = firstTime;
     for (var i = 0; i < quantity; i++) {
       plan.create(
-          "BiteBanana",
-          new DirectiveStart.Absolute(currentTime),
-          Map.of()
+          new NewDirective(
```

Comment:
Is naming an activity common enough that we should provide a `plan.create` overload?

Reply:
I don't think so